test_communication/package.xml

```diff
@@ -11,21 +11,23 @@
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>Apache License 2.0</license>
 
+  <build_depend>ament_cmake_auto</build_depend>
+  <build_depend>builtin_interfaces</build_depend>
+  <build_depend>rclcpp</build_depend>
+  <build_depend>rclpy</build_depend>
+  <build_depend>rcl</build_depend>
```

Comment:
I think that they should be build depends, so that the call to `ament_auto_find_build_dependencies()` will have the side effect of causing the subsequent call to `rosidl_generate_interfaces()` generating for all the languages.

But I could be very wrong; I've honestly never understood how to use the various `_depend` tags or how they interact with what happens in CMake :)